app/scripts/filters/resources.js

```diff
@@ -1084,7 +1084,18 @@ angular.module('openshiftConsole')
       return lastFinishTime;
     };
   })
+  // gets the first status condition.
   .filter('statusCondition', function() {
```

Comment:
Hm, you can't change the behavior of this filter since it's used elsewhere (bind dialog)